src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={`${
```

Comment:
the component accepts an array and ID and displays ui based on this data
the component should not expect that the data may be broken
responsibility for the correctness of the data lies with the person using the component
so replace to `className={classNames({ 'is-active': tab.id === selectedTabId })}`

Reply:
The tests don't pass otherwise, I tried >~<